backend/config/lightning.go

```diff
@@ -15,17 +15,35 @@
 
 package config
 
+import (
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/accounts/types"
+	"github.com/digitalbitbox/bitbox-wallet-app/util/jsonp"
+)
+
+// LightningAccountConfig is the configuration of a single Lightning account.
+type LightningAccountConfig struct {
+	// Mnemonic is the wallet node generated from the device entropy.
+	Mnemonic string `json:"mnemonic"`
+	// RootFingerprint is fingerprint of the keystore that generated the entropy.
+	RootFingerprint jsonp.HexBytes `json:"rootFingerprint"`
+	// Code is the code of the lightning account.
+	Code types.Code `json:"code"`
+	// Number is the lightning account incremental number.
+	Number uint16 `json:"number"`
+}
+
 // LightningConfig holds information related to the lightning config.
 type LightningConfig struct {
 	// Inactive is true if lightning has not yet been setup.
 	Inactive bool `json:"inactive"`
```

Comment:
is this flag still needed, now that the the code is setup for allowing more than one account?